lightning/config/global.go

```diff
@@ -115,7 +115,7 @@ func LoadGlobalConfig(args []string, extraFlags func(*flag.FlagSet)) (*GlobalCon
 	pdAddr := fs.String("pd-urls", "", "PD endpoint address")
 	dataSrcPath := fs.String("d", "", "Directory of the dump to import")
 	importerAddr := fs.String("importer", "", "address (host:port) to connect to tikv-importer")
-	backend := fs.String("backend", "", `delivery backend ("importer" or "mysql")`)
+	backend := fs.String("backend", "", `delivery backend ("importer" or "tidb")`)
```

Comment:
where we handle the `backend` config? Should we changed the logic about `backend` config?

Reply:
This is to fix a typo. The setting should be `tidb` instead of `mysql`.